src/content/integrations/astro-vtbot.md

```diff
@@ -4,6 +4,8 @@ title: "astro-vtbot"
 description: "The 👜 Bag of Tricks ✨ for Astro's View Transitions"
 categories:
   - "css+ui"
+  - "toolbar"
```

Comment:
Most of this data is scraped automatically from npm, so I think this will be wiped next time the scraper runs. If you want to have the toolbar category, you can add `"dev-toolbar"` to the `keywords` in your `package.json`.